completions/doas

```diff
@@ -0,0 +1,45 @@
+# doas(1) completion                                       -*- shell-script -*-
+
+_comp_cmd_doas()
+{
+    local cur prev words cword split
+    _init_completion -s || return
+
+    local i
+
+    for ((i = 1; i <= cword; i++)); do
+        if [[ ${words[i]} != -* ]]; then
+            local PATH=$PATH:/sbin:/usr/sbin:/usr/local/sbin
+            local root_command=${words[i]}
+            _command_offset $i
+            return
+        fi
+        [[ ${words[i]} == -@(!(-*)[uCLs]) ]] &&
```

Comment:
What's the difference between what you wrote and my suggestion below? They both seem to work, but my suggestion seems simpler.

```suggestion
        [[ ${words[i]} == -@([uCLs]) ]] &&
```